docs/bpmn2/cdp-poc.bpmn

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we use this or the one in resources?

Reply:
The one inside resources, this file has been deleted like the others

Reply:
update: I reported the process here for reference